components/base/kinematicbase/ptgKinematics.go

```diff
@@ -36,9 +36,11 @@ const (
 type ptgBaseKinematics struct {
 	base.Base
 	motion.Localizer
-	logger       logging.Logger
-	frame        referenceframe.Frame
-	ptgs         []tpspace.PTGSolver
+	logger logging.Logger
+	frame  referenceframe.Frame
+	// NOTE: What is the implementation we use for this PTGSolve when it comes to MoveOnGlobe requests?
+	ptgs []tpspace.PTGSolver
+	// NOTE: What is this mutext protecting?
```

Comment:
why does it not need to protect the base or any other members of this struct?

Reply:
`currentInput` is the only element that is changed by this struct; nothing else needs protection.

If the underlying, wrapped base needs mutexes it would implement those itself, not here.